### ISSUE TITLE
a question about unsupervised loss

### ISSUE THREAD

**Issue by Amos1109** [leofansq/CPCL#1]
`loss_w_1 = torch.max(F.softmax(logits_cons_tea_1, dim=-1))`
`loss_w_2 = torch.max(F.softmax(logits_cons_tea_2, dim=-1))`

in train.py ,line 181-182, the dim seem to be wrong?

**Comment by leofansq** [comment]
Thank you for pointing this out! We will check if we upload the wrong version for other experiments.
